app/lib/theme/colors.ts

```diff
@@ -0,0 +1,6 @@
+// BC GOV colors
+export const BC_GOV_LINKS_COLOR: string = "#1A5A96";
+export const BC_GOV_PRIMARY_BRAND_COLOR_BLUE: string = "#003366";
+// App specific colors
+export const TASKLIST_GREY_BG_COLOR: string = "#E5E5E5";
+export const MAIN_BG_COLOR: string = "#fafafc";
```

Comment:
Love this refactor. I'm not sure about the naming of LINKS and MAIN_BG_COLOR though--links is used on things besides links, and bg-color is used in the tasklist not just main. Maybe there are more accurate names? (Unless these are the BC design system names, in which case disregard.)

Reply:
The link is coming from BC GOV design system: https://developer.gov.bc.ca/Design-System/Colour-Palette
But for the other one I really didn't have a better name and I'm terrible at naming variables :D please suggest some!

Reply:
It is generally a good idea to listen to Chris Coyier about css ^